### PR TITLE
Security left nav reordering

### PR DESCRIPTION
**Pull request by bmckercher123**
Updated left nav for Security tab as part of Q2 content rewrite.

- Changes approved by Alex
- Includes renaming of existing Tutorials under a new section, Tutorials
- Includes removal of section "Host Endpoints" (moved in previous PR to Reference)
- Includes "fit & finish" consistency check and link fixes

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit fe0b657991f0bbbe22250060a054353fddd222c1

https://deploy-preview-2768--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit e1f101e38f6d01e18928d5f92868931c172a5580

https://deploy-preview-2768--calico.netlify.com

File: master/security/high-connection-workloads.md [hunk 1]
```diff
@@ -88,4 +88,4 @@ spec:
 ```
 ### Above and beyond
 
-[Calico GlobalNetworkPolicy](https://docs.projectcalico.org/v3.6/reference/calicoctl/resources/globalnetworkpolicy)
+[Global network policy](https://docs.projectcalico.org/v3.6/reference/calicoctl/resources/globalnetworkpolicy)
```
Comment (lmm) [inline master/security/high-connection-workloads.md]: We can use the site.baseurl and page.version variables here.

File: master/security/host-forwarded-traffic.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ An applyOnForward: true policy affects all traffic through the HEP (connections
 
 #### preDNAT policy
 
-Hosts are often configured to perform Destination Network Address Translation before forwarding certain packets. A common example of this in cloud computing is when the host acts as a reverse-proxy to load balance service requests for a set of backend workload instances. This How To guide on applying policy to [Kubernetes nodePorts]({{site.baseurl}}/{{page.version}}/security/kubernetes-node-ports) is a specific example of such a reverse-proxy.
+Hosts are often configured to perform Destination Network Address Translation before forwarding certain packets. A common example of this in cloud computing is when the host acts as a reverse-proxy to load balance service requests for a set of backend workload instances. A specific example of such a reverse-proxy is this How To guide on applying policy to [Kubernetes nodePorts]({{site.baseurl}}/{{page.version}}/security/kubernetes-node-ports).
```
Comment (lmm) [inline master/security/host-forwarded-traffic.md]: > A specific example of such a reverse-proxy is this How To guide on applying policy to

This sentence got mangled 😅

File: master/security/kubernetes-default-deny.md [hunk 3]
```diff
@@ -31,17 +31,17 @@ For other endpoint types (VMs, host interfaces), the default behavior is to deny
...
 
 ### How to
 
-Although you can use any of the following policies to create default deny policy for Kubernetes pods, we recommend using the Calico global network policy. A Calico global network policy applies to all workloads (VMs and containers) in all namespaces, as well as hosts (computers that run the hypervisor for VMs, or container runtime for containers). Using a Calico global network policy supports a conservative security stance for protecting resources.
+Although you can use any of the following policies to create default deny policy for Kubernetes pods, we recommend using the {{site.prodname}} global network policy. A {{site.prodname}}the hypervisor for VMs, or container runtime for containers). Using a {{site.prodname}} global network policy supports a conservative security stance for protecting resources.
```
Comment (lmm) [inline master/security/kubernetes-default-deny.md]: `A {{site.prodname}}the`

Comment (bmckercher123) [reply master/security/kubernetes-default-deny.md]: Fixed.

File: master/security/host-forwarded-traffic.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/host-forwarded-traffic.md]: Fixed.

File: master/security/high-connection-workloads.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: Fixed.

File: master/security/network-policy-openstack.md [hunk 4]
```diff
@@ -100,5 +100,5 @@ spec:
 
 ### Above and beyond
 
-- For additional Calico network policy features, see [Calico Network Policy]({{site.baseurl}}/{{page.version}}/reference/resources/networkpolicy) and [Calico Global Network Policy]({{site.baseurl}}/{{page.version}}/reference/resources/globalnetworkpolicy)
-- For details on the OpenStack integration with Calico, see [Calico for OpenStack]({{site.baseurl}}/{{page.version}}/networking/openstack/dev-machine-setup)
+- For additional {{site.prodname}}network policy features, see [{{site.prodname}} network policy]({{site.baseurl}}/{{page.version}}/reference/resources/networkpolicy) and [Calico global network policy]({{site.baseurl}}/{{page.version}}/reference/resources/globalnetworkpolicy)
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply master/security/network-policy-openstack.md]: Fixed.

File: master/security/tutorials/protect-hosts.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+the cluster provides various useful services that are exposed as Kubernetes
+NodePorts, i.e., as well-known TCP port numbers that appear to be available on
+any node in the cluster. The administrator does want to expose some
+of those NodePorts to traffic from outside.
+
+In this example we will use pre-DNAT policy applied to the external interfaces
+of each cluster node:
+
+- to disallow incoming traffic from outside, in general
+
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply master/security/tutorials/protect-hosts.md]: Is this the correct file? I search and cannot find these words on protect host

**Pull request merged**